### PR TITLE
fix(list): updated demo page

### PR DESCRIPTION
**Pull request by kara**
Quick PR to update demo page with new images and avatar styles.

r: @jelbourn

### CODE REVIEW

File: src/demo-app/list/list-demo.html [hunk 1]
```diff
@@ -24,6 +23,7 @@ <h3 md-line>{{contact.name}}</h3>
 
     <md-list>
       <md-list-item *ngFor="#message of messages">
+        <img md-list-avatar [src]="message.image">
```
Comment (jelbourn) [inline src/demo-app/list/list-demo.html]: nit: the images should have a filled-in `alt` attr

**Comment by jelbourn** [comment]
@kara tslint failed

**Comment by jelbourn** [comment]
LGTM once ci is green

**Comment by kara** [comment]
Fixed tslint and added alt text.

**Pull request merged**